up.go

```diff
@@ -453,22 +453,25 @@ func (e *Editor) kill() {
 	e.value = e.value[:e.cursor]
 }
 
-func (e *Editor) killWord() {
-	if e.cursor > 0 {
-		foundBeginingOfWord := false
-		cursor := e.cursor - 1
-		for !foundBeginingOfWord {
-			if cursor == 0 || (e.value[cursor] != ' ' && e.value[cursor-1] == ' ') {
-				foundBeginingOfWord = true
-			} else {
-				cursor--
-			}
+// unixWordRubout removes the part of the word on the left of the cursor. A word is
+// delimited by whitespaces.
```

Comment:
It's still kinda not super clear to me what they mean by whitespaces. Thanks to your pointer (big ❤️  for going the extra mile!), I took a look at [the manpage you suggested](https://linux.die.net/man/3/readline), and searching for "space" doesn't seem to lead to any definition, how lovely. Maybe the assumption is that this should be inferred from `LC_ALL` etc.? I'm pondering what else could show up here; some things that came to my mind are `\t` and maybe `\v`, as well as some unicode things like `&nbsp;` etc; but honestly I personally don't see it as an especially real concern in Latin scripts. [As to non-Latin ones](https://github.com/akavel/up/pull/42), I have no slightest idea. I'm thinking we could consider dropping [unicode.IsSpace](https://golang.org/pkg/unicode/#IsSpace) here and calling it a day, but I guess I'm also ok keeping it at `' '` for now; I'll defer the choice to you and accept it whichever of those you pick.

Reply:
Using `unicode.IsSpace` seems to makes sense, yes.